rust/agents/relayer/src/checkpoint_relayer.rs

```diff
@@ -161,11 +164,11 @@ impl CheckpointRelayer {
                     {
```

Comment:
Looking at the `get_messages_between` call a few lines above, do we need to change that implementation to something like this?

```
    /// Only gets the messages desinated for the Relayers inbox
    /// to_leaf_index is inclusive
    async fn get_messages_between(
        &self,
        from_leaf_index: u32,
        to_leaf_index: u32,
    ) -> Result<Option<Vec<CommittedMessage>>> {
        let mut messages: Vec<CommittedMessage> = vec![];
        let mut current_leaf_index = from_leaf_index;
        while current_leaf_index <= to_leaf_index {
// ....
```
(Note the `<` changed to `<=`)
I think atm it'll skip the message at index `signed_checkpoint_index`, which I don't think we want?

Reply:
Hmm not sure I'm following. The implementation in `get_messages_between` did change from < to <= no? The comment was still outdated though

Reply:
Oh yeah lol, I didn't realize `get_messages_between` was changed and must've been looking at my local branch to understand what was going on when I called this out. Looks good to me!